src/main/java/in/projecteka/consentmanager/user/PatientsController.java

```diff
@@ -111,7 +114,7 @@
                 .switchIfEmpty(Mono.error(ClientError.userNotFound()))
                 .flatMap(this::getVerfiedIdentifier)
                 .switchIfEmpty(Mono.error(ClientError.unAuthorized()))
-                .flatMap(verifiedIdentifier -> getGenerateOtpResponse(verifiedIdentifier, request.getUsername()))
+                .flatMap(verifiedIdentifier -> getGenerateOtpResponseFor(verifiedIdentifier, request.getUsername(), OtpAttempt.Action.OTP_REQUEST_RECOVER_PASSWORD,SendOtpAction.RECOVER_PASSWORD))
```

Comment:
What is sendOtp Action?

Reply:
It is to differentiate sending OTP according to different actions.

Reply:
Ok. Are we using this same method for generating OTPs for multiple flows.

Reply:
yeah. So previously patients/generateotp was only calling it. but now to send the OTP for recovering CMID also, we are calling this method.

Reply:
Ok.